Source/UZKFileInfo.h

```diff
@@ -78,6 +78,9 @@ typedef NS_ENUM(NSInteger, UZKCompressionMethod) {
  */
 @property (readonly) UZKCompressionMethod compressionMethod;
 
-
+/**
+ @brief posixPermissions of the file.
```

Comment:
Please replace with a brief explanation of how to read the permissions (which NSNumber property to read, whether it's a bit mask, etc.